package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk",
-  "version": "3.15.0",
+  "version": "3.15.1",
```

Comment:
Since the API surface is expanding, should this necessitate a minor bump?

Reply:
oh yeah good call